src/features/projectsV2/ProjectsMap/ProjectSiteDropDown.tsx

```diff
@@ -0,0 +1,103 @@
+import { useState, useMemo, useCallback } from 'react';
+import { useTranslations } from 'next-intl';
+import SiteIcon from '../../../temp/icons/SiteIcon';
+import styles from './SiteDropdown.module.scss';
+import DropdownUpArrow from '../../../temp/icons/DropdownUpArrow';
+import DropdownDownArrow from '../../../temp/icons/DropdownDownArrow';
+import { SetState } from '../../common/types/common';
+import ProjectSiteList from './microComponents/ProjectSiteList';
+import { area } from '@turf/turf';
+import { Feature, MultiPolygon, Polygon } from 'geojson';
+
+export interface SiteProperties {
+  lastUpdated: {
+    date: string;
+    timezone: string;
+    timezone_type: number;
+  };
+  name: string;
+  description: string | null;
+  id: string;
+  status: string | null;
+}
+
+export type ProjectSite =
+  | Feature<Polygon | MultiPolygon, SiteProperties>[]
+  | undefined
+  | null;
+
+interface Props {
+  projectSites: ProjectSite;
+  selectedSite: number;
+  setSelectedSite: SetState<number>;
+}
+
+const ProjectSiteDropdown = ({
+  projectSites,
```

Comment:
I suggest grouping the components and styles for `ProjectSiteDropDown` in one folder, similar to the `MapFeatureExplorer` and `ProjectPopup`.